### ISSUE TITLE
Require Linux/ARM64 Docker Image for azcopy 

### ISSUE THREAD

**Issue by odidev** [PeterDaveHello/docker-azcopy#2]
I have tried using peterdavehello/azcopy docker image on Linux/ARM64 platform but there is no arm64 tag available in the docker hub. 
 I have successfully built the docker image on my Linux/ARM64 Environment after editing ‘GOARCH: arm64’ using the below command: 
~~~
 docker build --build-arg AZCOPY_VERSION=10.11.0 -t azcpy64:latest . 
~~~
I have checked the Travis-CI but there is no release process mentioned in the same and I could not find any other release process to push the image to the Dockerhub .  

It would be helpful if ARM64 Docker Image for peterdavehello/azcopy will be available. 

May I know, what is the release process used to push Docker Image to the Docker hub? 
If required, I am happy to contribute and raise PR.

**Comment by PeterDaveHello** [comment]
Hi @odidev,

Sorry for the late reply, I was using Docker Hub's auto build before, but currently, they stopped the feature for the free tier accounts, I've sent my application to the Docker Hub Open Source program for months, multiple times, but no any response yet, so the "current" process is to build the image on my workstation and push to Docker Hub, contribution to have multi-arch build for Travis CI and local build are welcome, thanks for your contribution!

**Comment by PeterDaveHello** [comment]
Update: I've published a test build with the latest azcopy v10.12.2, please help test it on your platform.

https://hub.docker.com/layers/peterdavehello/azcopy/test/images/sha256-e0ec84482d7d9d6a2bf6d0b2487cd3e00965258d3564dc604120373e9526b77e?context=explore

**Comment by PeterDaveHello** [comment]
Done in edc169bb2b14b79fcee777c23ced61450649f3ef
